models/image_classification/imagenet_preprocess.py

```diff
@@ -0,0 +1,19 @@
+import mxnet
+from mxnet.gluon.data.vision import transforms
+
+# Pre-processing function for ImageNet models
+def preprocess(img):   
+    '''
+    Preprocessing required on the images for inference with mxnet gluon
+    The function takes path to an image and returns processed tensor
+    '''
+    transform_fn = transforms.Compose([
```

Comment:
similarly, could you use numpy to implement these transformation functions?

It won't be very hard.

Reply:
@houseroad same as my comment above.